tyk-stack/values.yaml

```diff
@@ -189,6 +189,12 @@ global:
   # Choose the storageType for Tyk. [ "mongo", "postgres" ]
   storageType: &globalStorageType postgres
 
+  # hashKeys specifies that if your Tyk Gateway is using hashed keys, set this value to true so it matches.
+  # The Dashboard will now operate in a mode that is compatible with key hashing.
+  # This takes precedence over the dashboard value i.e `dashboard.hashKeys`
+  # It is used to set TYK_DB_HASHKEYS
```

Comment:
It will enable hashkeys in both gateway and dashboard. Can you update the comment accordingly?